interview-questions.md

```diff
@@ -8,50 +8,73 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a inputed value, index, or number in a function
+such as (array[bean, fruit, meat], /string"Hello How are you", index[2], 25) 
+where a argument an argument is if a value is given under a condition and then 
+it is logged in result as a return
+
+Researched answer: Parameters are the names listed in the functions Definition
+                    Example : function example(paramenter) {
+                        console.log(parameter)
+                    }
+                    An Argument are real values passed to the function
+                    const argument = "LEARN"
```

Comment:
We can think of a parameter as a placeholder for when we are given the data to pass into the function, or the argument. Good job including an example!